pkg/templates/stamper.go

```diff
@@ -196,51 +195,52 @@ func (s *Stamper) applyYtt(ctx context.Context, template string) (*unstructured.
 	log := logr.FromContextOrDiscard(ctx)
 
 	// limit execution duration to protect against infinite loops or cpu wasting templates
-	ctx, cancel := context.WithTimeout(ctx, 1*time.Second)
+	_, cancel := context.WithTimeout(ctx, 1*time.Second)
```

Comment:
my understanding is that we're keeping the PR as a draft because we can't pass this context down to `RunWithFiles` - is that right?

Reply:
Yup that's right. I opened an issue in ytt for this: https://github.com/vmware-tanzu/carvel-ytt/issues/593